azavea/raster-foundry
# Feature/increase scene grid resolution

## Overview

This commit replaces the current scene grid endpoint/implementation with
one that uses Leaflet's grid layer to request summary aggregate
information for each tile on the map. This speeds up the endpoint by
reducing the amount of information required to power it.

The grid layer itself uses Konva to draw rectangles and style
them. Canvas events are used to display the count when the canvas
rectangle is hovered over. Additional styling work is necessary, but
this gives us hopefully a start.

### Checklist

~- [ ] Styleguide updated, if necessary~

### Demo

![screenshot from 2017-01-03 13 16 23](https://cloud.githubusercontent.com/assets/898060/21619829/6f828e38-d1bf-11e6-8069-6eff8f23da66.png)

### Notes

Styling will need to be updated, but I think this first implementation is passable for now.

## Testing Instructions

 * Switch to branch, run migrations via `./scripts/setup`
 * Start server and application front end
 * Go to browse page, ensure scene grid shows up

Closes #812
